products/bootstrap.md

```diff
@@ -70,5 +75,6 @@ Versions in _Maintenance_ should not have any changes landed, except for:
 - **Important** documentation updates
 
 Unless a change is urgent, _Maintenance_ releases are likely to be made with minimal frequency.
-The `v4` branch is currently in Active LTS and will receive bug fixes till 2021-07-01, after which
-it will only receive critical fixes till its End of Life.
+Maintenance status will expire on the dates listed in the table above (middle column) and the version will reach End of Life, where no more updates will be published.
```

Comment:
```suggestion
```

You are right, we should not have documented information already available in the release table. I propose to remove the entire sentence.

Reply:
Agreed. I deleted that paragraph here [198ec39](https://github.com/endoflife-date/endoflife.date/pull/4859/commits/198ec39ef5e895a5f3a2298a18294eddcae54747)